paddle/fluid/framework/op_registry.h

```diff
@@ -67,10 +67,35 @@ struct OperatorRegistrar : public Registrar {
 
 class OpRegistry {
  public:
+  /**
+   * @brief Return an OperatorBase constructed by using type, inputs, outputs,
+   *        attrs.
+   *        In dygraph mode, inputs, output, attrs wil be set to empty map to
+   *        to improve the execution efficiency of dygraph.
+   *        Dygraph mode will using:
+   *        framework::OpRegistry::CreateOp(type, {}, {}, {}, false).
+   *
+   * @param[str] type          The operator type.
+   * @param[map] inputs        Inptus map of the operator.
+   * @param[map] outputs       Outputs map of the operator.
+   * @param[unordered_map]     Attributes map of thee operator.
+   * @param[bool] attr_check
+   *            Whether do the attribute check before OperatorBase construction.
+   *            Default is true.
+   *            Attr_check is used to control the check of attribute map.
+   *            The check of attribute map have two purpose:
+   *            1. check attribute item is valid.
```

Comment:
check whether the attribute item is valid or not.

Reply:
done